.github/workflows/CI-CD_SUGCON_NA.yml

```diff
@@ -0,0 +1,39 @@
+name: CI-CD - SUGCON NA
+on: 
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+    paths:
+      - .github/workflows/CI-CD_SUGCON_NA.yml
+      - .github/workflows/build_NextJs.yml
+      - .github/workflows/deploy_vercel.yml
+      - 'src/Project/Sugcon/SugconNaSxa/**'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - .github/workflows/CI-CD_SUGCON_NA.yml
+      - .github/workflows/build_NextJs.yml
+      - .github/workflows/deploy_vercel.yml
+      - 'src/Project/Sugcon/SugconNaSxa/**'
+
+jobs:
+
+  build-na-site:
+    if: github.ref != 'refs/heads/main'
+    uses: ./.github/workflows/build_NextJs.yml
+    with:
+      workingDirectory: ./src/Project/Sugcon/SugconNaSxa
+
+  deploy-na-site:
+    uses: ./.github/workflows/deploy_vercel.yml
+    needs: build-na-site
+    if: always() && 
+        (github.repository_owner == 'Sitecore' && ((github.event.pull_request.head.repo.full_name == github.repository) || (github.event_name == 'push'))) &&
+        needs.build-staging-anz-site.result != 'failure' &&
+        needs.build-staging-anz-site.result != 'cancelled' && 
+        needs.build-production-anz-site.result != 'failure' &&
+        needs.build-production-anz-site.result != 'cancelled'
```

Comment:
These lines should reference the renamed `build-na-site` job.

Looks like we need to update the others as well as we no longer have split staging & production jobs. You just need two lines checking that your na job  isn't failed or cancelled.

Reply:
Fixed this for NA. 

And actually found EU was referencing anz jobs. But was this ever working since it was referencing jobs that no longer existed?

Reply:
I would have worked, I just would never have failed.

We'll get NA in and working, then I can tidy up the rest seprately.